### ISSUE TITLE
Missing aria label identifier for main view

### ISSUE THREAD

**Issue by GVodyanov** [nextcloud/contacts#5046]
### Steps to reproduce

1. Go to main section (the part without the nav sidebar)

### Expected behavior

Have an aria label

### Actual behavior

No label

### PHP engine version

None

### Web server

None

### Database

None
